### PR TITLE
KAFKA-12408: Document omitted ReplicaManager metrics

### PR DESCRIPTION
**Pull request by dongjinleekr**
I found it while configuring a JMX monitoring.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by dongjinleekr** [comment]
Retest this please.

**Comment by dongjinleekr** [comment]
@ableegoldman @dajac @tombentley Could you have a look? If fixes errors in JMX documentation.

File: docs/ops.html [hunk 1]
```diff
@@ -1576,12 +1576,17 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
       <tr>
         <td>Partition counts</td>
         <td>kafka.server:type=ReplicaManager,name=PartitionCount</td>
-        <td>mostly even across brokers</td>
+        <td>The number of partitions assigned to the broker.</td>
```
Comment (tombentley) [inline docs/ops.html]: The 3rd column is the normal value, not a description, so "mostly even across brokers" is actually correct.

File: docs/ops.html [hunk 2]
```diff
@@ -1576,12 +1576,17 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
...
+      </tr>
+      <tr>
+        <td>Offline Replica counts</td>
+        <td>kafka.server:type=ReplicaManager,name=OfflineReplicaCount</td>
+        <td>The number of offline partitions assigned to the broker.</td>
```
Comment (tombentley) [inline docs/ops.html]: The normal value of this one should be zero.

File: docs/ops.html [hunk 3]
```diff
@@ -1576,12 +1576,17 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
...
       </tr>
       <tr>
         <td>Leader replica counts</td>
         <td>kafka.server:type=ReplicaManager,name=LeaderCount</td>
-        <td>mostly even across brokers</td>
+        <td>The number of leader partitions assigned to the broker.</td>
```
Comment (tombentley) [inline docs/ops.html]: Again, "mostly even across brokers" is correct.

File: docs/ops.html [hunk 4]
```diff
@@ -1596,6 +1601,11 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
         <td>kafka.server:type=ReplicaManager,name=IsrExpandsPerSec</td>
         <td>See above</td>
       </tr>
+      <tr>
+        <td>Failed ISR update rate</td>
+        <td>kafka.server:type=ReplicaManager,name=FailedIsrUpdatesPerSec</td>
+        <td>The rate of new ISR update failure.</td>
```
Comment (tombentley) [inline docs/ops.html]: 0

**Comment by dongjinleekr** [comment]
Hi @tombentley

Thanks for pointing out my mistakes. Here is the update.

**Review by tombentley — APPROVED** [review]

**Pull request merged**